src/main/assembly/dist/docs/examples/ddm/example1.xml

```diff
@@ -21,7 +21,7 @@
     xmlns:dc="http://purl.org/dc/elements/1.1/"
     xmlns:dct="http://purl.org/dc/terms/"
     xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
-    xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/md/ddm/ http://easy.dans.knaw.nl/schemas/md/2017/09/ddm.xsd">
+    xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/2018/05/ddm.xsd">
```

Comment:
Some observations; could you verify that these are correct?

You can only test the latest-before-one version, right? As long as you haven't released the newest versions of this scheme, you can't test it using `example1.xml`. Hence, a change in the DDM schema and updating this example (including a validation test) would require (1) a PR with the schema changes, (2) a release to `easy01` of this project for the new schema, (3) a PR with the adjusted `example1.xml` file (4) a release to `easy01` of this project for the examples to be up to date.

If this is true, it makes me wonder if automated testing in this project is really worth it.